GraphQL/InputMapper/SearchQueryMapper.php

```diff
@@ -87,62 +46,45 @@ function ($sortClauseClass) {
     }
 
     /**
-     * @param array $queryArg
-     * @param $dateMetadata
-     * @return \eZ\Publish\API\Repository\Values\Content\Query\Criterion\DateMetadata[]
+     * @param string $inputField
+     * @return bool
      */
-    private function mapDateMetadata(array $queryArg = [], $dateMetadata)
+    private function existsCriteriaMapperForField(string $inputField) : bool
     {
-        if (!isset($queryArg[$dateMetadata]) || !is_array($queryArg[$dateMetadata])) {
-            return [];
-        }
-
-        $targetMap = [
-            'Created' => Query\Criterion\DateMetadata::CREATED,
-            'Modified' => Query\Criterion\DateMetadata::MODIFIED,
-        ];
+        return isset($this->criteriaMappers[$inputField]);
+    }
 
-        if (!isset($targetMap[$dateMetadata])) {
-            echo "Not a date metadata\n";
-            return [];
-        }
+    /**
+     * @param $inputValue
+     * @return array
+     */
+    private function mapSortByPart($inputValue) {
```

Comment:
Remember to typehint return values as well: `private function mapSortByPart($inputValue): array`. Also, The opening curly bracket for methods gets its own line (I gotta setup CS check).